src/main/webapp/script.js

```diff
@@ -73,7 +169,7 @@ function displayVisualization(censusDataArray, title) {
 }
 
 // Takes in a 2D array from the census API and displays the visualization
-function drawRegionsMap(censusDataArray, title) {
+function drawRegionsMap(censusDataArray) {
```

Comment:
I think this function should be drawRegionsMap(censusDataArray, title) because you still use the title in the function